Scripts/Sources/Automation/Shell.swift

```diff
@@ -0,0 +1,113 @@
+import Foundation
+
+class Shell {
+    static func executeAndPrint(_ command: String, timeout: Double = 10, allowFailure: Bool = false) throws {
+        print("$ \(command)")
+        let output = try executeShell(commandPath: "/bin/bash", arguments: ["-c", command], timeout: timeout, allowFailure: allowFailure) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tevelee/Eval/blob/fc973589729700ac4c55183b359a3f3047c6157f/Scripts/Sources/Automation/Shell.swift#L6"></span>Line should be 120 characters or less: currently 139 characters</td>
    </tr>
  </tbody>
</table>